### PR TITLE
Savva Dzhumaliev Tasks 91-100

### PR DESCRIPTION
**Pull request by EasileleM**

### CODE REVIEW

File: tasks/91_make-a-person/make-a-person.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export class Person {
+  constructor(fullName) {
+    const fullNameSplit = fullName.split(' ');
+    let firstName = fullNameSplit[0];
```
Comment (vovcyan) [inline tasks/91_make-a-person/make-a-person.js]: Please use ES6 feature for it

File: tasks/92_calculate-orbital-period/calculate-orbital-period.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * @param {Object[]} bodies
+ * @return {Object[]}
+ */
+export function orbitalPeriod(bodies) {
+  const GM = 398600.4418;
+  const EARTH_RADIUS = 6367.4447;
+  const PI = 3.14159265359;
+  const bodiesCopy = JSON.parse(JSON.stringify(bodies));
+  for (const body of bodiesCopy) {
+    body['orbitalPeriod'] = Math.round(2 * PI * Math.sqrt(Math.pow(body.avgAlt
```
Comment (vovcyan) [inline tasks/92_calculate-orbital-period/calculate-orbital-period.js]: Please split line

Comment (vovcyan) [inline tasks/92_calculate-orbital-period/calculate-orbital-period.js]: body.orbitalPeriod ?

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: tasks/96_greatest-common-divisor/greatest-common-divisor.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/**
+ * greatestCommonDivisor - return greatest common divisor
+ * of two passed numbers
+ * @param {number} a
+ * @param {number} b
+ * @return {number}
+ */
+export function greatestCommonDivisor(a, b) {
```
Comment (akimovaa) [inline tasks/96_greatest-common-divisor/greatest-common-divisor.js:8]: The best solution! Congrats.

**Review by akimovaa — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**